### PR TITLE
v2.6 rc-1

### PR DESCRIPTION
**Pull request by brouillette**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -408,19 +416,30 @@ clean-all:
...
+		cd lib/gohan &&  \
+		$(MAKE) clean-api ; \
+	fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
+
+
+	@# Skip triggering top level makefile stop for both public and gohan
+	@if [[ $* != public && $* != gohan ]]; then \
+		echo "-- Stopping $* --" ; \
+		docker-compose stop $* &> tmp/logs/${EXECUTED_NOW}/$*/clean.log ; \
+	fi &>> tmp/logs/${EXECUTED_NOW}/$*/clean.log
```
Comment (ppillot) [inline Makefile]: This would break on MacOS

File: README.md [hunk 2]
```diff
@@ -178,7 +178,12 @@ make docker-secrets
 # Build base images
 make build-common-base
 
-# And open up the cluster's gateway with
+# If you are running the bentoV2 with the use of an external identity provider, i.e setting BENTOV2_USE_EXTERNAL_IDP=1, run both
+make run-gateway
+# and
+make auth-setup
+
+# Otherwise, only open the cluster's gateway with
```
Comment (ppillot) [inline README.md:186]: It seems contradictory with the next line. Should it be `make run-gateway` instead?

**Review by ppillot — COMMENTED** [review]
Non critical possible issues after a cursory look at the PR. Please consider checking this before merge.

File: README.md (hunk 2 above)
Comment (brouillette) [reply README.md:186]: I see the confusion.
This is correct, in that `auth-setup` in the second case still does perform important configuration, despite the command name, that need be done before spinning up the gateway container (see how the nginx.conf template differs in both cases). The main difference is that when running with a local auth provider like KC, it requires the gateway in its base default setting to be up and running so the one of the automatic scripts can wire them together.

File: Makefile (hunk 1 above)
Comment (brouillette) [reply Makefile]: because of the `&>>` ?

Comment (ppillot) [reply Makefile]: Yes. We have patched this by `>>` `2>&1` in all the places

File: README.md (hunk 2 above)
Comment (ppillot) [reply README.md:186]: Thanks for the clarification!

File: Makefile (hunk 1 above)
Comment (brouillette) [reply Makefile]: as this is not critical to the impending deployment in production, I'd say we elaborate on this next round on `qa/2.6`

**Pull request merged**